avocado/utils/pmem.py

```diff
@@ -302,16 +277,17 @@ def enable_namespace(self, namespace="all", region="", bus="", verbose=False):
         """
         args = namespace
         if region:
-            args = f"{args} -r {region}"
+            args += f" -r {region}"
         if bus:
-            args = f"{args} -b {bus}"
+            args += f" -b {bus}"
         if verbose:
-            args = f"{args} -v"
+            args += " -v"
 
-        if process.system(
-            f"{self.ndctl} enable-namespace {args}", shell=True, ignore_status=True
-        ):
+        cmd = f"{self.ndctl} enable-namespace {args}"
+
+        if os.system(cmd) != 0:
```

Comment:
I wouldn't recommend usage of  `os.system` instead of `process.system` because the `os.system` won't log any outputs to stderr or stdout of that command, and we might lose important information, especially if it used with `-v` flag.

Reply:
agree @richtja

Reply:
Thank you for your guidance. It has been fixed